keyboards/gray_studio/apollo80/readme.md

```diff
@@ -0,0 +1,18 @@
+# APOLLO 80
+
+80% / TKL keyboard with a 5.5 degree angle ran by Airpotter. PCB Manufactured by DEMO Studio and Keyboard manufactured by Gray Studio.
+
+-   Keyboard Maintainer: [Chris Beswick](https://github.com/ChrisBeswick/)
+-   Hardware Supported: Stock / Default Apollo80 PCB
+-   Hardware Availability: [Geekhack GB](https://geekhack.org/index.php?topic=110033.0)
+-   Group Buy Location: [Graystudio.club](https://graystudio.club/products/gb-space80-apollo)
+
+How to build this firmware using QMK MSYS
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```